mpqp/execution/devices.py

```diff
@@ -204,3 +204,36 @@ def from_arn(arn: str):
             if elem.value in arn:
                 return elem
         raise ValueError(f"No device found for ARN `{arn}`.")
+
+
+class GOOGLEDevice(AvailableDevice):
+    """Enum regrouping all available devices provided by CIRQ."""
+
+    CIRQ_LOCAL_SIMULATOR = "LocalSimulator"
+    PROCESSOR_RAINBOW = "rainbow"
+    PROCESSOR_WEBER = "weber"
+    IONQ_SIMULATOR = "simulator"
+    IONQ_QPU = "qpu"
+
+    def is_remote(self):
+        if self.name.startswith("IONQ"):
+            return True
+        return False
+
+    def is_ionq(self):
+        return self.name.startswith("IONQ")
+
+    def is_gate_based(self) -> bool:
```

Comment:
Is this function really useful ? Are we handling other devices than gate_based for the moment ?

Reply:
It's the same thing for AWSDevice, I see there's the same function that return true also. so I got it because maybe it's a function used somewhere.

Reply:
![image](https://github.com/ColibrITD-SAS/mpqp/assets/133856040/ecc511bb-6fb1-4507-a72a-f97039ef265e)
I see that it is only used in tests, so I think we can remove them safely from both ?

Reply:
Is this test really useful by the way ? I'm not sure about it ^^"
it seems like a mix of implementation details and over testing (over testing is when you ask "what is this test telling you?", and you don't have a great answer)
I might be wrong in this.
Also, why `assert a is True` and `assert b is False` instead of `assert a` and `assert not b` ?